subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -455,8 +455,8 @@ uint8_t ll_adv_params_set(uint16_t interval, uint8_t adv_type,
 
 #if defined(CONFIG_BT_CTLR_PRIVACY)
 	adv->own_addr_type = own_addr_type;
-	if (adv->own_addr_type == BT_ADDR_LE_PUBLIC_ID ||
-	    adv->own_addr_type == BT_ADDR_LE_RANDOM_ID) {
```

Comment:
Are the old values used for other purposes? if not, delete their definitions?

Reply:
Out of scope of this PR, but `BT_ADDR_LE_PUBLIC_ID` and `BT_ADDR_LE_RANDOM_ID` are not Bluetooth address types. I think they are shoehorned here from some HCI event definition, and if that's the case, they should go with the other HCI definitions.